app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,6 @@
 <resources>
     <string name="app_name">ENGAGEHF</string>
```

Comment:
We might actually want to adjust the app name as well?

Reply:
updated [here](https://github.com/StanfordSpezi/SpeziKt/pull/177/commits/8427b7f7d23ad95ae3f30206c37e093eb2af949c) 👍

Reply:
Thank you!